Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/Artifact/Factory/TestImpactTestEnumerationSuiteFactory.cpp

```diff
@@ -84,4 +87,72 @@ namespace TestImpact
             return testSuites;
         }
     } // namespace GTest
+    namespace Python
+    {
+
+        AZStd::vector<AZStd::pair<AZStd::string, AZStd::vector<TestEnumerationSuite>>> TestEnumerationSuitesFactory(
+            const AZStd::string& testEnumerationData)
+        {
+            const char newLineDelim = '\n';
+            size_t startLine;
+            size_t endLine = 0;
+
+            AZStd::vector<AZStd::pair<AZStd::string, AZStd::vector<TestEnumerationSuite>>> pairs =
+                AZStd::vector<AZStd::pair<AZStd::string, AZStd::vector<TestEnumerationSuite>>>();
```

Comment:
You can just do `AZStd::vector<AZStd::pair<AZStd::string, AZStd::vector<TestEnumerationSuite>>> pairs;`